api/asyncapi.yaml

```diff
@@ -20,23 +20,22 @@ channels:
       description: Send messages to the API
       message:
         oneOf:
-          - $ref: '#/components/messages/objectAdded'
+          - $ref: '#/components/messages/newChecksum'
```

Comment:
it's good that we stick to the asyncAPI, but I really wonder if anyone will ever use that for reference...

Reply:
Yes, the NewChecksum model was not generated for some reason.. then I realizer it was missing this line